### PR TITLE
Navigationbar

### PR DESCRIPTION
**Pull request by desplodur**

### CODE REVIEW

**Review by mbosselmann — COMMENTED** [review]
Good work! 👍 I only have nit-picking remarks about removing some code and a svg.

While exploring your app, I saw that you gave your question components a fixed width, which is why they are currently not responsive. Maybe it would be better to change `width` to `max-width`. Just an extra aspect that I haven't noticed before and that you should keep in mind. :)

File: src/components/Header.js [hunk 1]
```diff
@@ -1,13 +1,3 @@
...
-	return (
-		<header>
-			<nav>
-				<Link to="/">Home</Link>
-				<Link to="/questions">Questions</Link>
-				<Link to="/profile">Profile</Link>
-			</nav>
-		</header>
-	);
+	return <header></header>;
```
[the remark this answers is not in the document]
Comment (desplodur) [reply src/components/Header.js:2]: Sure :)

File: src/components/navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+						cy="25.5"
+						r="24.5"
+						fill="white"
+						stroke="black"
+						strokeWidth="2"
+					/>
+					<path d="M26 15V36" stroke="black" />
+					<line x1="36" y1="25.5" x2="15" y2="25.5" stroke="black" />
+				</svg>
+			</Link>
```
[the remark this answers is not in the document]
Comment (desplodur) [reply src/components/navbar/index.js:43]: nope :)

File: src/components/navbar/styled.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+const StyledNavBar = styled.nav``;
```
[the remark this answers is not in the document]
Comment (desplodur) [reply src/components/navbar/styled.js:3]: For now i´ve always created a basic component setup. Therefore ive added the styled component, basic testing and almost a useless stories file.

[the remark this answers is not in the document]
Comment (desplodur) [reply src/components/navbar/styled.js:3]: For this particular PR i´ve added a background color in the component styling. In the next week i will implement a basic styling. :)

[the remark this answers is not in the document]
Comment (mbosselmann) [reply src/components/navbar/styled.js]: Hm. The last PR I checked, was your first PR where you had basic styling and a test. :) Not needed files should not be part of your PR and therefore be removed (as they do nothing) or filled with necessary code. 👍

**Review by mbosselmann — APPROVED** [review]
I approve 👍

**Pull request merged**